### PR TITLE
Automated PR for Spencer-86

### PR DESCRIPTION
**Pull request by myndzi**

### CODE REVIEW

File: src/Spencer-86.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          <b>2.</b>
+          <p>The origin of this software must not be misrepresented, either by explicit claim or by omission.</p>
+        </li>
+        <li>
+          <b>3.</b>
+          <p>Altered versions must be plainly marked as such, and must not be misrepresented as being the
+             original software.</p>
+        </li>
+      </list>
+      <p>Beware that some of this code is subtly aware of the way operator precedence is structured in regular
```
Comment (sam-ellis) [inline src/Spencer-86.xml]: This paragraph seems non-legal in nature and could be made optional.

Comment (jlovejoy) [inline src/Spencer-86.xml]: good point, it does not appear on the examples on Fedora either.  Should it be optional or simply removed?

Comment (bradleeedmondson) [inline src/Spencer-86.xml]: agree that it's non-legal.

can we run a google-fu search to see whether it's included in source in the wild? if not, we can remove; if so, we can make optional

Comment (pmadick) [inline src/Spencer-86.xml]: Agree with Brad's conclusions.

Comment (jlovejoy) [inline src/Spencer-86.xml]: to remove, discussed on 8/16 legal call

**Comment by sam-ellis** [comment]
Pull request submitted.

**Pull request merged**